activerecord/lib/active_record/enum.rb

```diff
@@ -119,8 +132,9 @@ def serialize(value)
       attr_reader :name, :mapping
     end
 
-    def enum(definitions)
+    def enum(definitions, options = {})
```

Comment:
Either prefix (and presumably suffix) will have to go as possible enum names, or we'll need to use double hashes. I'd prefer to go the double hash route. It's slightly uglier, but the point of this feature is to reduce the number of reserved words. In which case it'd be `def enum(definitions, prefix: false, suffix: false)`